src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt

```diff
@@ -113,15 +112,15 @@ object SectionSyncListener : Listener {
         val section = block.location.section ?: return
         val section2 = block.location.correspondingSection ?: return
 
-        if(
+        if (
             block.location.inSectionOverlap
-            && section.isOnTopOf(section2)
+//            && section.isOnTopOf(section2)
             && block.blockData !is Levelled // Water / Lava
         ) isCancelled = true
 
-        if(!section.isOnTopOf(section2)) {
-            block.sync()
-        }
+//        if(!section.isOnTopOf(section2)) {
```

Comment:
Couldn't we just not register this handler instead of all of this commenting?

Reply:
We still want some of the functionality, and this was a temp "fix" until i fixed the bug

Reply:
Right, the listener itself is still valid just not this handler. Sounds good. Just fix the monitor level handler above and its good.